### PR TITLE
Cli predict

### PR DESCRIPTION
**Pull request by cbartz**
Add prediction mode to command line interface

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -88,7 +88,9 @@ docker run --rm -it -v \
 ## Usage
 
 ### Input data 
-In order to train or evaluate a model you have to provide data. 
+In order to train a model, evaluate a model or predict the quality you have to
```
Comment (annakasprzik) [inline README.md:91]: "predict the quality of its output"

File: README.md [hunk 2]
```diff
@@ -100,6 +102,8 @@ where
 (this is basically the output of the MLC method)
 - ``true_labels`` is a comma-separated list of true labels (ground truth)
 
+Note that you can omit the ``true_labels`` section if you only want to predict the quality of the data.
```
Comment (annakasprzik) [inline README.md:105]: "quality of the output"? otherwise will be confused with training data?

File: README.md [hunk 3]
```diff
@@ -114,6 +118,9 @@ This is a  folder with three files per document:
 * ``doc.tsv`` : ground truth
 * ``doc.txt`` : document content
 
+As above, you may omit the ``doc.tsv`` if you just want to
+predict the quality of the data.
```
Comment (annakasprzik) [inline README.md:122]: "output"

File: qualle/interface/cli.py [hunk 4]
```diff
@@ -245,6 +263,32 @@ def cli_entrypoint():
     )
     rest_parser.set_defaults(func=handle_rest)
 
+    predict_parser = subparsers.add_parser(
+        'predict',
+        description='Predict the quality for a a collection of automated '
```
Comment (annakasprzik) [inline qualle/interface/cli.py:268]: "a a" --> "a"

File: qualle/interface/cli.py [hunk 5]
```diff
@@ -245,6 +263,32 @@ def cli_entrypoint():
...
+                    'subject indexing (MLC) results.'
+    )
+    predict_parser.set_defaults(func=handle_predict)
+
+    predict_parser.add_argument(
+        'predict_data_path', type=Path,
+        help='Path to data. '
+             'Accepted are either a tsv file or a folder in annif format. '
```
Comment (annakasprzik) [inline qualle/interface/cli.py:276]: "Accepted inputs are", "Annif"

**Review by annakasprzik — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (cbartz) [reply README.md:91]: The quality of an MLC result is predicted. Therefore it is not the output of the model itself, but the output of an MLC model.
Perhaps rephrase to: "predict the quality of an MLC result"

Comment (annakasprzik) [reply README.md:91]: okay -- as long as "the quality of what" is specified

File: README.md (hunk 2 above)
Comment (cbartz) [reply README.md:105]: Perhaps be more precise: "quality of the MLC output"?

Comment (annakasprzik) [reply README.md:105]: okay

**Pull request merged**